src/pages/settings/Settings.tsx

```diff
@@ -131,39 +141,34 @@ const Settings: FC = () => {
             href={`${docBaseLink}/server/`}
             title="Learn more about server configuration"
           >
-            Settings
+            Server settings
```

Comment:
I wonder if this is just more confusing for the user? I preferred the old version. It also matches better the entry in the side navigation.

Reply:
True, especially with the UI setting in it, this is better with the shorter version.